src/components/Comment/Comment.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const Comment = ({ name, body, email }) => (
+  <>
+
+    <h3>
+      Title:
+      {' '}
+      {' '}
+      {name}
+    </h3>
```

Comment:
```suggestion
    <h3>{`Title:  ${name}`}</h3>
```
використовуй string template